code/game/gamemodes/antag_spawner.dm

```diff
@@ -70,3 +70,83 @@
 	ticker.mode.update_synd_icons_added(R.mind)
 	R.mind.special_role = "syndicate"
 	R.faction = list("syndicate")
+
+/obj/item/weapon/antag_spawner/slaughter_demon //Warning edgiest item in the game
+	name = "vial of blood"
+	desc = "A magically infused bottle of blood, distilled from countless murder victims. Used in unholy rituals to attract horrifying creatures."
+	icon = 'icons/obj/wizard.dmi'
+	icon_state = "vial"
+	var/checking = 0
+
+
+/obj/item/weapon/antag_spawner/slaughter_demon/attack_self(mob/user as mob)
+	//var/list/demon_candidates = get_candidates(BE_ALIEN)
+	if(user.z != ZLEVEL_CENTCOMM)//this is to make sure the wizard does NOT summon a demon from the Den..
```

Comment:
Should be `if(user.z == ZLEVEL_CENTCOMM)`, currently it won't let you spawn them anywhere except Z level 2.